src/dependency-manager/spec/resource-spec.ts

```diff
@@ -1,59 +1,9 @@
-import { IsObject, IsOptional, IsString, ValidateNested } from 'class-validator';
+import { IsOptional, ValidateNested } from 'class-validator';
```

Comment:
Remove the changes to:
- docs/5-reference/2-architect-yml.md
- src/dependency-manager/schema/architect.schema.json

These files are auto-generated.

Reply:
reverted changes to the files:
* docs/5-reference/2-architect-yml.md
* src/dependency-manager/schema/architect.schema.json